_config/site.yml

```diff
@@ -98,6 +98,7 @@ profiles:
     js_minifier: disabled
     html_minifier: disabled
     legacy_post_syntax_highlighting: disabled
+    ignore_older_than_days: 30
```

Comment:
nice one. Do you know whether one can pass in (aka override) these values form the command line. In development mode I often want to see a bit more and would love to change this more dynamically.